scripts/send-bundles-sizes-differences.js

```diff
@@ -0,0 +1,181 @@
+    const sign = diff.percentageChange > 0 ? '+' : ''
+    message += `| ${formatName(diff.name)} | ${baseSize} | ${localSize} | ${sign}${diff.percentageChange}% |\n`
+  })
+
+  return message
+}
+
+async function updateOrAddComment(difference, resultsBaseQuery, resultsLocalQuery, prNumber, commentId) {
+  const message = createMessage(difference, resultsBaseQuery, resultsLocalQuery)
+  const method = commentId ? 'PATCH' : 'POST'
+  const payload = {
+    pr_url: `https://github.com/DataDog/browser-sdk/pull/${prNumber}`,
+    message,
+    header,
+    org: 'DataDog',
+    repo: 'browser-sdk',
+  }
+  const response = await fetch('https://pr-commenter.us1.ddbuild.io/internal/cit/pr-comment', {
+    method,
+    headers: {
+      Authorization: `Bearer ${githubAuthToken}`,
+    },
+    body: JSON.stringify(payload),
+  })
+  if (!response.ok) {
+    throw new Error(`HTTP Error Response: ${response.status} ${response.statusText}`)
+  }
+}
+
+async function getCommentId(prNumber) {
+  const response = await fetch(`https://api.github.com/repos/DataDog/browser-sdk/issues/${prNumber}/comments`, {
+    method: 'GET',
+    headers: {
+      Authorization: `token ${GH_TOKEN}`,
+    },
+  })
+  if (!response.ok) {
+    throw new Error(`HTTP Error Response: ${response.status} ${response.statusText}`)
+  }
+  const comments = await response.json()
+  const targetComment = comments.filter((comment) => comment.body.startsWith('## Bundles Sizes Evolution'))[0]
+  if (targetComment !== undefined) {
+    return targetComment.id
+  }
+}
+
+function loadQuery(commitSha) {
+  return [
+    {
+      name: 'bundles_sizes_rum',
+      queries: `avg:bundle_sizes.rum{commit:${commitSha}}`,
+    },
+    {
+      name: 'bundles_sizes_logs',
+      queries: `avg:bundle_sizes.logs{commit:${commitSha}}`,
+    },
+    {
+      name: 'bundles_sizes_rum_slim',
+      queries: `avg:bundle_sizes.rum_slim{commit:${commitSha}}`,
+    },
+    {
+      name: 'bundles_sizes_worker',
+      queries: `avg:bundle_sizes.worker{commit:${commitSha}}`,
+    },
+  ]
+}
+
+async function queryWithRetry(budget, retries = 4) {
+  const now = Math.floor(Date.now() / 1000)
+  const date = now - 30 * 24 * 60 * 60
+  for (let i = 0; i < retries; i++) {
+    const response = await fetch(
+      `https://api.datadoghq.com/api/v1/query?query=${budget.queries}&from=${date}&to=${now}`,
+      {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+          'DD-API-KEY': getOrg2ApiKey(),
+          'DD-APPLICATION-KEY': getOrg2AppKey(),
+        },
+      }
+    )
+    if (!response.ok) {
+      throw new Error(`HTTP Error Response: ${response.status} ${response.statusText}`)
+    }
+    const data = await response.json()
+    if (data.series && data.series.length > 0 && data.series[0].pointlist && data.series[0].pointlist.length > 0) {
+      return {
+        name: budget.name,
+        size: data.series[0].pointlist[0][1],
+      }
+    }
+    await timeout(5000)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏What about extracting a constant for that? [#magic-number](https://en.wikipedia.org/wiki/Magic_number_(programming))